iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/GeneratorImpl.java

```diff
@@ -140,7 +140,7 @@ public TerminalExt getRegulatingTerminal() {
 
     @Override
     public GeneratorImpl setRegulatingTerminal(Terminal regulatingTerminal) {
-        ValidationUtil.checkRegulatingTerminal(this, (TerminalExt) regulatingTerminal, getNetwork());
+        ValidationUtil.checkRegulatingTerminal(this, regulatingTerminal, getNetwork());
```

Comment:
Why do you remove the (TerminalExt) cast?

Reply:
`checkRegulatingTerminal` has changed its signature: from `checkRegulatingTerminal(Validable, TerminalExt, Network)` to `checkRegulatingTerminal(Validable, Terminal, Network)` (cf. #1125). Hence, casting as `TerminalExt` is not necessary anymore.

Reply:
Ok, thanks for the explanation!